cmake/opentitan.cmake

```diff
@@ -0,0 +1,146 @@
+# Copyright 2024 ETH Zurich and University of Bologna.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Viviane Potocnik <vivianep@iis.ee.ethz.ch>
+
+include(ExternalProject)
+
+set(OPENTITAN_DIR ${CMAKE_SOURCE_DIR}/drivers/opentitan)
+set(SPARSE_CHECKOUT_FILE ${CMAKE_CURRENT_BINARY_DIR}/sparse-checkout)
+
+if(NOT DEFINED OPENTITAN_COMMIT_HASH)
+  message(FATAL_ERROR "Please set a commit hash in your target.")
+endif()
+
+message(STATUS "[OT Drivers] Using OpenTitan commit hash: ${OPENTITAN_COMMIT_HASH}")
```

Comment:
NIT, but please make sure the message aligns with the other ones. 
Also, I would use a `[CHIMERA-SDK]` tag to be consistent and avoid polluting the messages. This way all messages from the SDK can be easily spotted.